cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -67,6 +69,19 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             cy.get('div div').should('not.have.class', 'lazyload-placeholder');
           });
       });
+
+      if (serviceHasInlineLink(service)) {
+        it('should have an inline link in a paragraph that results in a successful navigation', () => {
+          cy.get('main > div > div > p > a[href]')
```

Comment:
```suggestion
          cy.get('main p a[href]')
```
I _think_ this should work, and be safer

Reply:
It works, but safer in what way? It's more relaxed certainly.